docs/getting-started/policies/application-types.rst

```diff
@@ -246,6 +273,52 @@ using the :ref:`operator-new` operator if the class has been **registered**:
         We must pass positional arguments to the class constructor because
         JavaScript does not support keyword arguments.
 
+    .. group-tab:: Rust
+        We can register a Rust struct or enum using ``Oso::register_class``.
+        ``register_class`` takes as input a ``Class``, which can be constructed
+        either using the ``#[derive(PolarClass)]`` proc-macro, or manually using
+        ``Class::new::<T>()``:
+
+        .. code-block:: rust
+            :caption: :fab:`rust` main.rs
+
+            #[derive(Clone, PolarClass)]
+            struct User {
+                #[polar(attribute)]
+                name: String,
+                #[polar(attribute)]
+                is_admin: bool,
+            }
+
+            impl User {
+                fn new(name: String, is_admin: bool) -> Self {
+                    Self { name, is_admin }
+                }
+
+                fn is_called_alice(&self) -> bool {
+                    self.name == "alice"
+                }
+            }
+
+            oso.register_class(
+               User::get_polar_class_builder()
+                    .set_constructor(User::new)
```

Comment:
Does this work? I was having trouble using constructors that took arguments without a wrapper lambda but that was a few days ago.

Reply:
Will try!

Reply:
Constructor worked, but a different example was incorrect.